demos/serverless/src/handlers.js

```diff
@@ -208,3 +208,105 @@ exports.end = async(event, context, callback) => {
   }).promise();
   callback(null, response);
 };
+
+exports.logs = async (event, context) => {
+
+  console.log(event);
+  var sequenceToken = "";
+  const logGroupName = "BrowserLogs";
+  var response = {
+    "statusCode": 200,
+    "headers": {},
+    "body": '',
+    "isBase64Encoded": false
+  };
+  try {
+    if (typeof JSON.parse(event.body) === 'undefined' || typeof JSON.parse(event.body).logs === 'undefined') {
+            response.body = "Empty";
+      return response;
+    } else {
```

Comment:
nit: I don't think you need an else here. You are already returning from the if statement.